test.js

```
@@ -10,3 +10,45 @@ test(t => {
 	t.is(m.getName(143), 'Snorlax');
 	t.is(m.getId('Snorlax'), 143);
 });
+
+test('English names', t => {
+	t.is(m.getName(1), 'Bulbasaur');
+	t.is(m.getName(100), 'Voltorb');
+	t.is(m.getName(200), 'Misdreavus');
+	t.is(m.getName(300), 'Skitty');
+	t.is(m.getName(400), 'Bibarel');
+	t.is(m.getName(500), 'Emboar');
+	t.is(m.getName(600), 'Klang');
+	t.is(m.getName(700), 'Sylveon');
+	t.is(m.getName(721), 'Volcanion');
+});
+
+test('German names', t => {
+	t.is(m.getName(1, 'de'), 'Bisasam');
+	t.is(m.getName(100, 'de'), 'Voltobal');
+	t.is(m.getName(200, 'de'), 'Traunfugil');
+	t.is(m.getName(300, 'de'), 'Eneco');
+	t.is(m.getName(400, 'de'), 'Bidifas');
+	t.is(m.getName(500, 'de'), 'Flambirex');
+	t.is(m.getName(600, 'de'), 'Kliklak');
+	t.is(m.getName(700, 'de'), 'Feelinara');
+	t.is(m.getName(721, 'de'), 'Volcanion');
+});
+
+test('Indices can be used with language codes', t => {
+	t.is(m.getId('Snorlax'), 143);
+	t.is(m.getId('Relaxo', 'de'), 143);
+});
+
+test('Language fallback is English', t => {
+	const snorlaxId = 143;
+	const snorlaxEnglish = 'Snorlax';
+
+	t.is(m.getId(snorlaxEnglish), snorlaxId);
+	t.is(m.getId(snorlaxEnglish, 'en'), snorlaxId);
+	t.is(m.getId(snorlaxEnglish, 'invalid'), snorlaxId);
+
+	t.is(m.getName(snorlaxId), snorlaxEnglish);
+	t.is(m.getName(snorlaxId, 'en'), snorlaxEnglish);
+	t.is(m.getName(snorlaxId, 'invalid'), snorlaxEnglish);
```

If you supply an invalid language code it should throw a useful error instead.